common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -240,8 +256,12 @@ def create_event(self, knowledge_node_uid, student_uid, event_type, **data):
         payload.update(data)
 
         # Send request
-        response = requests.post(url, headers=self.request_headers, data=payload)
-        event = json.loads(response.content)
+        try:
+            event = {}
```

Comment:
@macornwell This should go into the [`except` block](https://github.com/open-craft/edx-platform/pull/69#discussion_r148499568).